docs/AboutUs.md

```diff
@@ -30,7 +30,7 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ### Lan Jingbo
 
-<img src="images/Lan Jingbo.png" width="200px">
+<img src="images/lanjingbo.png" width="200px">
```

Comment:
Should you name the .png based on your github username instead (Lan-Jingbo) (same with your /docs/team/.md)?